Aptoide/aptoide-client-v8
# [feature/ASV-1710] show background caption color given in the ws

**What does this PR do?**

   show background caption color given in the editorial ws response. default is orange

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] CaptionBackgroundPainter.java.java

**How should this be manually tested?**

  Home > check editorial content card caption
  Aptoide > Editorial > check cards caption
  Aptoide > Click an editorial content card > check feature graphic caption 

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1710](https://aptoide.atlassian.net/browse/ASV-1710)

**Code Review Checklist**
